Update moves.js

Mewthy opened the pull request:
Spa to Spd: Power Trick

Mewthy committed:
Update moves.js

Spa to Spd: Power Trick

DeathlyPlays commented on data/moves.js:
```diff
@@ -12364,21 +12364,33 @@ let BattleMovedex = {
 				this.add('-start', pokemon, 'Power Trick');
 				let newatk = pokemon.storedStats.def;
 				let newdef = pokemon.storedStats.atk;
+				let newspa = pokemon.storedStats.spd;
```
Let's move this into a mod folder for swapping powers

DeathlyPlays committed:
Relocate to mod folder

DeathlyPlays committed:
Swapping Powers: Modify Power Trick

Power Trick will now switch Special Attack and Special Defense along with the past effects.

DeathlyPlays committed:
Formats: Apply Mod to Swapping Powers
